components/camel-file/src/main/java/org/apache/camel/component/file/FileOperations.java

```diff
@@ -408,6 +411,16 @@ public boolean storeFile(String fileName, Exchange exchange, long size) throws G
         }
     }
 
+    private void writeFileFromString(String body, File target) throws IOException {
```

Comment:
Can you rename to `writeFileByString` and move the method down to where the others are